<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1520</width>
    <height>816</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>470</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1520</width>
    <height>816</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>风机桨叶智能巡检系统</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>find.png</normaloff>find.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="play_label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>103</width>
            <height>68</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1035</width>
            <height>683</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="3">
           <widget class="QLineEdit" name="url_le">
            <property name="minimumSize">
             <size>
              <width>600</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>600</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>视频源:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5">
           <widget class="QPushButton" name="gather_btn">
            <property name="text">
             <string>采集</string>
            </property>
           </widget>
          </item>
          <item row="0" column="7">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="6">
           <widget class="QPushButton" name="stop_btn">
            <property name="text">
             <string>停止</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>30</height>
             </size>
            </property>
            <property name="text">
             <string>当前检测机组编号：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="machineNum_le">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>190</width>
              <height>28</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>190</width>
              <height>28</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="turbine_table">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>420</width>
            <height>305</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>机组编号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>检测状态</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>检测日期</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="defect_table">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>420</width>
            <height>305</height>
           </size>
          </property>
          <column>
           <property name="text">
            <string>序号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>缺陷类型</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>面积(㎡)</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>直径(m)</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>距轮毂(m)</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="minimumSize">
           <size>
            <width>418</width>
            <height>72</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>418</width>
            <height>72</height>
           </size>
          </property>
          <property name="title">
           <string>操作界面</string>
          </property>
          <widget class="QWidget" name="horizontalLayoutWidget_5">
           <property name="geometry">
            <rect>
             <x>20</x>
             <y>20</y>
             <width>391</width>
             <height>51</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="pushButton_11">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>自动拼接</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_12">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>损伤检测</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="save_btn">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>35</height>
               </size>
              </property>
              <property name="text">
               <string>保存记录</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_fillinfo"/>
   <addaction name="action_manage"/>
   <addaction name="action_querylog"/>
   <addaction name="action_fanstitching"/>
   <addaction name="action_genreport"/>
   <addaction name="action_curactionend"/>
   <addaction name="action_curinfo"/>
  </widget>
  <action name="action_fillinfo">
   <property name="text">
    <string>填写信息</string>
   </property>
  </action>
  <action name="action_manage">
   <property name="text">
    <string>管理检测计划</string>
   </property>
  </action>
  <action name="action_querylog">
   <property name="text">
    <string>查询检测记录</string>
   </property>
  </action>
  <action name="action_genreport">
   <property name="text">
    <string>生成检测报告</string>
   </property>
  </action>
  <action name="action_curactionend">
   <property name="text">
    <string>终止当前操作</string>
   </property>
  </action>
  <action name="action_curinfo">
   <property name="text">
    <string>当前信息</string>
   </property>
  </action>
  <action name="action_fanstitching">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>叶片拼接</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
